packages/frontend-2/components/project/page/acc/Tab.vue

```diff
@@ -0,0 +1,211 @@
+        color="outline"
+        size="sm"
+        @click="tokens = undefined"
+      >
+        Log out
+      </FormButton>
+      <div v-if="tokens?.access_token" class="flex flex-row items-center space-x-2">
+        <FormButton
+          class="mr-2"
+          hide-text
+          :icon-left="DocumentDuplicateIcon"
+          color="outline"
+          @click="copy(tokens?.access_token)"
+        >
+          Copy to clipboard
+        </FormButton>
+        {{ tokens?.access_token.slice(0, 32) }}...
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+// import { useQuery, useSubscription } from '@vue/apollo-composable'
+// import { projectAccSyncItemsQuery } from '~/lib/acc/graphql/queries'
+// import { onProjectAccSyncItemUpdatedSubscription } from '~/lib/acc/graphql/subscriptions'
+import type { AccTokens, AccUserInfo } from '~/lib/acc/types'
+import { DocumentDuplicateIcon } from '@heroicons/vue/24/outline'
+
+const props = defineProps<{ projectId: string }>()
+const { triggerNotification } = useGlobalToast()
+const { copy } = useClipboard()
+
+const apiOrigin = useApiOrigin()
+const tokens = ref<AccTokens>()
+const hasTokens = computed(() => !!tokens.value?.access_token)
+const loadingTokens = ref(true)
+const userInfo = ref<AccUserInfo>()
+const loadingUser = ref(false)
+
+// const { result: accSyncItemsResult, refetch: refetchAccSyncItems } = useQuery(
+//   projectAccSyncItemsQuery,
+//   () => ({
+//     id: props.projectId
+//   })
+// )
+
+// const accSyncItems = computed(
+//   () => accSyncItemsResult.value?.project.accSyncItems.items || []
+// )
+
+// const { onResult: onProjectAccSyncItemsUpdated } = useSubscription(
+//   onProjectAccSyncItemUpdatedSubscription,
+//   () => ({
+//     id: props.projectId,
+//     itemIds: accSyncItems.value.map((s) => s.accFileLineageUrn)
+//   })
+// )
+
+// onProjectAccSyncItemsUpdated((res) => {
+//   refetchAccSyncItems()
+//   triggerNotification({
+//     type: ToastNotificationType.Info,
+//     title: 'Acc Sync Item updated',
+//     description: res.data?.projectAccSyncItemsUpdated.accSyncItem?.accFileLineageUrn
+//   })
+// })
+
+// const syncs = ref<AccSyncItem[]>([
+//   {
+//     id: '1',
+//     projectId: '',
+//     modelId: '',
+//     projectName: 'test',
+//     modelName: 'test',
+//     status: 'paused',
+//     createdBy: 'Oguzhan Koral',
+//     accItem: {
+//       id: 'yo',
+//       attributes: {
+//         name: 'whatever.rvt',
+//         displayName: 'whatever.rvt'
+//       }
+//     }
+//   }
+// ])
+
+// AUTH + TOKEN FLOW
+const fetchTokens = async () => {
+  try {
+    const res = await fetch(`${apiOrigin}/api/v1/acc/auth/status`, {
+      credentials: 'include'
+    })
+    if (!res.ok) return
+    tokens.value = await res.json()
+  } finally {
+    loadingTokens.value = false
+  }
+}
+fetchTokens()
+
+const authAcc = async () => {
+  try {
+    const response = await fetch(`${apiOrigin}/api/v1/acc/auth/login`, {
```

Comment:
ideally/in the future fetches should use something like useFetch/useAsyncData for caching, and i'd probably also store it in a singleton state (useState()) so that we can ask for it from different places without having to re-do the fetching